src/LeanCode.ContractsGenerator/Generation/ContractsGenerator.cs

```diff
@@ -29,14 +29,15 @@ public Export Generate(List<Export> externalContracts, bool excludeExternalContr
         }
 
         merged.ProjectName = export.ProjectName;
+        merged.Protocol = Protocol.Max([export.Protocol, .. externalContracts.Select(e => e.Protocol)]);
```

Comment:
This is too simplistic (I guess) - merging contracts v1 and v2 might be impossible.

Reply:
Is https://github.com/leancodepl/contractsgenerator/blob/4820d8dead9fb18179c01adc369afdb6e2b4fdae/src/LeanCode.ContractsGenerator/Protocol.cs#L107-L110 not enough to guard against it?

Reply:
:see_no_evil: